app/pandas.ipynb

```diff
@@ -145,18 +329,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Age\"] > 18\n",
```

Comment:
The current code only returns a boolean series indicating which rows have 'Age' greater than 18. To filter the DataFrame, you should use: `df[df['Age'] > 18]`.